### ISSUE TITLE
DataLoader cannot use multithreading on windows？

### ISSUE THREAD

**Issue by tongyuhome** [qfgaohao/pytorch-ssd#55]
Hi，Thank you very much for sharing。
I have a problem. When I run the code on the Windows10 system, I always get an error when setting ‘num_workers’ to a number >0.
Report an error：“ Can't pickle local object 'TrainAugmentation.__init__.<locals>.<lambda>' ”
Can you give me some solutions or suggestions?
Thank you

**Comment by Qasim-Abdul-Majeed** [comment]
You can passed '--num_workers 0' as a command line argument.
It will work using 01 thread or not exploiting multi threading.

**Comment by tongyuhome** [comment]
> You can passed '--num_workers 0' as a command line argument.
> It will work using 01 thread or not exploiting multi threading.

Thanks for replying, using single thread is very slow, I just want to use multithreading, how do I need to modify it?

**Comment by andreistirb** [comment]
in order to use multithreading in windows you should try changing the lambda transformation into a function (callable class, to be more specific).
Something like this:
`class NormalizesStd(object):
    def __init__(self, std):
        self.std = np.array(std, dtype=np.float32)
    def __call__(self, image, boxes=None, labels=None):
        image = image.astype(np.float32)
        image /= self.std
        return image.astype(np.float32), boxes, labels `

**Comment by gangstarappa1** [comment]
Hi, i'm having the same problem and would like to use multithreading. Can someone here show an example of how to implement these lines:

class NormalizesStd(object): def __init__(self, std): self.std = np.array(std, dtype=np.float32) def __call__(self, image, boxes=None, labels=None): image = image.astype(np.float32) image /= self.std return image.astype(np.float32), boxes, labels

Thank you very much!
